public/js/utils.jsx

```diff
@@ -41,6 +41,11 @@ export function gettext(string){
 }
 
 
+export function setTitle(string) {
```

Comment:
Perhaps add a basic test for this. Maybe inject the document object that way you can check the property is set. You can now use default args `setTitle(string, doc=document)`.